packages/grafana-ui/src/components/VizLegend/VizLegendListItem.tsx

```diff
@@ -107,6 +107,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
     fontSize: 'inherit',
     padding: 0,
     userSelect: 'text',
+    margin: theme.spacing(0.5, 0),
```

Comment:
similar here, think a better fix is to remove this margin and change `paddingLeft` to `padding` on both lines 102 and 109 here: https://github.com/grafana/grafana/blob/eledobleefe/a11y-focus-outline-73390/packages/grafana-ui/src/components/VizLegend/VizLegendList.tsx